tensorboard/default.py

```diff
@@ -86,4 +87,30 @@ def get_plugins():
 
   :rtype: list[Union[base_plugin.TBLoader, Type[base_plugin.TBPlugin]]]
   """
+
   return _PLUGINS[:]
+
+# TODO(stephanwlee): Combine with get_plugins when concept of first-party
+# plugins is undone.
+def get_dynamic_plugins():
+  """Returns a list specifying TensorBoard's dynamically loaded plugins.
+
+	A dynamic TensorBoard plugin is specified using entry_points [1]. This will be
+  the de facto mechanism to load plugins and there no longer will be a concept
```

Comment:
It’s not just the _de facto_ mechanism; it’s the actual mechanism. (In
fact, one might say that the _de facto_ mechanism is the legacy one,
given that the new one is not yet used.)